dashboard/app/models/levels/karel.rb

```diff
@@ -122,6 +122,9 @@ def toolbox(type)
     <category name="Bee">
       <block type="maze_nectar"></block>
       <block type="maze_honey"></block>
+      <block type="bee_n_honey"></block>
+      <block type="bee_n_honey"></block>
```

Comment:
Should one of those be `nectar`?

Reply:
whoops, yes good catch thanks!

Reply:
fixed:
![image](https://user-images.githubusercontent.com/8787187/114911140-8b8f0b80-9dd3-11eb-9954-4dce02fad7ed.png)